app/views/idv/capture_doc/back_image.html.slim

```diff
@@ -8,7 +8,7 @@ h1.h3.my0 = t('doc_auth.headings.upload_back')
   .clearfix.mxn1
     .sm-col.sm-col-8.px1.mt2
       = f.input :image, label: false, as: 'file', required: true
-  p.red.bold = flow_session[:error_message]
+  p.red.bold == flow_session[:error_message]
```

Comment:
an alert class we can use here is 
```
.alert.alert-error == flow_session[:error_messsage]
```

Though as I commented on the other line we may want to style the list of suggestions separately (possibly in a regular blue info alert) or something else in which case there would be a different class here.